test-crates/duchess-java-tests/tests/java-to-rust/rust-libraries/native_fn_arrays.rs

```diff
@@ -27,3 +29,17 @@ fn break_bytes(num: i64) -> duchess::Result<Java<java::Array<i8>>> {
 
     return Ok(java_array);
 }
+
+#[duchess::java_function(java_to_rust_arrays.JavaArrayTests::fillWithOnes)]
+fn fill_with_ones(arr: Option<&mut duchess::java::Array<i8>>, len: i32) -> i64 {
```

Comment:
it looks like the API supports having this be `&T`? can you add a ui test that presumably fails if you don't do that?